demo/app.js

```diff
@@ -153,6 +153,7 @@ const Tabs = ({ tabValues, manifestUrl }) => {
       <TabPanel id="transcriptsTab" tabId="transcripts" tabIndex={1} activeTab={activeTab}>
         <Transcript
           playerID="iiif-media-player"
+          showSearch={true}
```

Comment:
As I understood this prop can be used to turn the search functionality on/off.
I couldn't find where this gets read into the code in the transcript component to be used. Could you please point me to the place where this gets used?

Reply:
Ah, I forgot to update that line actually. Originally I had the various search props spread out on the `Transcript`
component like so: 

```jsx
<Transcript searchEnabled={true} matcherFactory={myCustomMatcher} />
```

Just before I did the PR did some cleanup. Now all the search things go inside a single prop: `search`
```jsx
<Transcript search={false} />
```
```jsx
<Transcript search={{ matchesOnly: true, matcherFactory: myCustomMatcher }} />
```

Reply:
Could you please update the code to reflect the changes? Thanks!

Reply:
[1b144a4](https://github.com/samvera-labs/ramp/pull/497/commits/1b144a413cd5de34911be49a72c12aa3f53a667a) cleans that up